cedar-policy-core/src/validator/types.rs

```diff
@@ -1121,14 +1098,6 @@ pub enum EntityRecordKind {
     /// Attributes in this case are not stored inline but must be looked up in
     /// the schema, based on the elements of the [`EntityLUB`].
     Entity(EntityLUB),
-
-    /// We special case action entities, which store their attributes directly, like `Record`s do
-    ActionEntity {
-        /// Type name of the action entity
-        name: EntityType,
-        /// Attributes of the action entity
-        attrs: Attributes,
-    },
```

Comment:
Checking my understanding, since we're removing `attrs`, there's no longer a reason to special-case action entities and previous users of `ActionEntity` will just use `Entity` instead?